generic_processing.conf

```diff
@@ -39,20 +39,16 @@
 [FILTER]
    Name modify
    Match  kube.*
-   Rename apm-qa/projectname _tag_projectName
    Rename snappyflow/component _tag_componentName
-   Rename apm-qa/appname _tag_appName
```

Comment:
can u add like Rename ${SFAPM_PROJECTNAME_LABEL} _tag_projectName

Reply:
Changes done in following commit